### ISSUE TITLE
Pixel color function doesn't work on device

### ISSUE THREAD

**Issue by qmchenry** [qmchenry/UILint#20]
For at least some devices, the guard that bitsPerPixel is 32 fails because there are, in fact, 64 bits per pixel on the wide gamut devices. This may be fixed as part of a wider issue around computing contrast ratios. It may only be necessary to get luminance from the colors which may simplify this.
